packages/manager/modules/pci/src/projects/project/instances/instance/stop/stop.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class PciInstanceStopController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    Poller,
```

Comment:
**thought:** Services injection could be sorted.

Reply:
On my dev, i always put the angular dependencies first and project deps. after.
I sorted like i explain.

Reply:
Done.